packages/replayio/src/utils/recordings/getRecordings.ts

```diff
@@ -26,7 +26,9 @@ export function getRecordings(processGroupIdFilter?: string): LocalRecording[] {
 
           Object.assign(recording.metadata, metadata);
 
-          const { argv, process, processGroupId, uri } = metadata;
+          const { argv, process, processGroupId, title, uri } = metadata;
+
+          recording.metadata.title = title;
```

Comment:
I'm not sure if that's a valid scenario right now but it doesn't seem unreasonable to have multiplied `addMetadata` entries for a single recording. In such a case, if the title would be in the first one then we'd remove it when processing the second one.
```suggestion
          if (title) {
            recording.metadata.title = title;
          }
```